server/game/cards/08-MotC/AsahinaTakamori.js

```diff
@@ -0,0 +1,31 @@
+const DrawCard = require('../../drawcard.js');
+const { CardTypes, Durations, Players } = require('../../Constants');
+const AbilityDsl = require('../../abilitydsl');
+
+class AsahinaTakamori extends DrawCard {
+    setupCardAbilities() {
+        this.reaction({
+            title: 'Choose a character to pacify',
+            when: {
+                onCardPlayed: (event) => event.card.type === CardTypes.Character && event.card.isFaction('crane')
+            },
+            target: {
+                controller: Players.Opponent,
+                cardType: CardTypes.Character,
+                cardCondition: (card, context) => card.getCost() <= context.event.card.getCost(),
```

Comment:
You should use `card.costLessThan(num)` to deal with cards with no cost being immune to these effects

Reply:
Changed.